File: github-mvi-android/remote/src/release/java/com/rightpoint/mvi/example/remote/NetworkSettings.java
package com.rightpoint.mvi.example.remote;

import dagger.Module;
import dagger.Provides;
import okhttp3.logging.HttpLoggingInterceptor.Level;

@Module
public abstract class NetworkSettings {
    @Provides
    static Level providesLoggingLevel() {
        return Level.NONE;
    }
}

File: github-mvi-android/remote/src/debug/java/com/rightpoint/github/mvi/remote/NetworkSettings.kt
package com.rightpoint.github.mvi.remote

import dagger.Module
import dagger.Provides
import okhttp3.logging.HttpLoggingInterceptor.Level

@Module
object NetworkSettings {
    @Provides
    @JvmStatic
    fun providesLoggingLevel(): Level {
        return Level.BODY
    }
}